cli/ops/websocket.rs

```diff
@@ -58,11 +79,7 @@ pub async fn op_ws_create(
   _bufs: BufVec,
 ) -> Result<Value, AnyError> {
   let args: CreateArgs = serde_json::from_value(args)?;
-  {
-    let s = state.borrow();
-    s.borrow::<Permissions>()
-      .check_net_url(&url::Url::parse(&args.url)?)?;
-  }
```

Comment:
@crowlKats please leave this block, but replace `?` with `.expect("Permission check should have been done in op_ws_check_permission")` and leave a comment about why we're having additional op for this purpose